wenv

```diff
@@ -314,14 +314,17 @@ OPTIONS
     shift $((OPTIND-1))
 
     [[ -z "$1" ]] && return 1
-    local wenv="$1"
 
     # TODO: generalize/clean this up
     if [[ $flag_d -eq 1 ]]; then
-        cat =(perl -pe "s|WENV_DIR=.*?$|WENV_DIR=\"`pwd`\"|" "$template") > "$WENV_CFG/wenvs/$wenv"
+        [[ -z "$2" ]] && echo "directory path and wenv name arguments are required with the -d flag" && return 1
+        local wenv_dir="$1"
+        local wenv="$2"
+        cat =(perl -pe "s|WENV_DIR=.*?$|WENV_DIR=\"`echo $wenv_dir`\"|" "$template") > "$WENV_CFG/wenvs/$wenv"
```

Comment:
Don't `echo $wenv_dir`, just put `$wenv_dir` in directly (i.e. `...WENV_DIR=$wenv_dir...`). The entire replacement command being in double quotes (`"s|...|...|"`) should expand it.